api/v1alpha1/edgedevice_types.go

```diff
@@ -378,6 +381,27 @@ type SystemVendor struct {
 	Virtual bool `json:"virtual,omitempty"`
 }
 
+type HostDevice struct {
+
+	// path
+	Path string `json:"path,omitempty"`
+
+	// Device type
+	DeviceType string `json:"deviceType,omitempty"`
+
+	// owner id
+	UID int32 `json:"owner,omitempty"`
```

Comment:
i can just assume that uint32 will be a better fit for this one and GID and version parts.
however, I don't recall why haven't we used it in other places when it made sense.

Reply:
my guess is to avoid mapping each value from int32 to int64. swagger generates only int64. 
I will change it since I'm making already a conversion from int32 -> int64